### ISSUE TITLE
Bad quality flipper conent

### ISSUE THREAD

**Issue by passa1248** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#693]
Hi!
As the title I have used material flipper but the content inside it it's very bad quality (as my english)
solutions?
Thanks in advance!

Simone

**Comment by ButchersBoy** [comment]
How bad?  :)

This is now in 3D...there is some compromise...

Can you post a sample?

**Comment by passa1248** [comment]
I don't know if it is the 3D effect but every content seems to be a bit blurred.
![00screen](https://cloud.githubusercontent.com/assets/19681865/25190746/87aaa146-252d-11e7-8889-f97711c33463.png)

**Comment by Keboo** [comment]
Looks to me like a [ClearType](https://msdn.microsoft.com/en-us/library/ms749295.aspx) issue. If you zoom way in on the one on the right you will notice the little bits of red and blue around the glyphs. You might try setting the [ClearTypeHint](https://blogs.msdn.microsoft.com/text/2010/01/14/cleartypehint/).
```XAML
RenderOptions.ClearTypeHint="Enabled"
```

This issue is likely caused because of the DropShadowEffect that is being applied ([docs](https://msdn.microsoft.com/en-us/library/ms748273.aspx))

**Comment by passa1248** [comment]
not work sorry :(

**Comment by Maradaehne** [comment]
I am running in a similiar issue, when i have two contents with a different size. The first smaller one is displayed normal, I flip and the second is also displayed normal. When I flip it back to the first one the control is resized but the content is enlarged not fitting into the control anymore.

**Comment by passa1248** [comment]
so the problem is only with 2 face with different size?

**Comment by Aangbaeck** [comment]
I have the exact same problem. Did you find a way to fix this? RenderOptions does not help.

**Comment by Aangbaeck** [comment]
I solved this by going to graphic card settings and disabling Morphologic filtering. It's clearly a settings issue. Now everything looks sharp again.

**Comment by glen-nicol** [comment]
I am getting similar artifacts in the transitioner after pages have finished animating. Specifically, I get a kind of vertical wave where the text in the transitioner blurs and then becomes clear repeatedly. I am using the WPF default font (Segoe UI) and font size 14 which renders just fine without the transitioner wrapping it. 

I have tried other fonts (verdana, Segoe UI Semilight, Calibri) which have less of an effect but changing fonts isn't really an option. It matches the fonts of other related apps. I have tried various combinations of TextOptions and RenderOptions with no changes.

I imagine it is the canvas or whatever makes the animation possible that is introducing the floating point variation in rendering. Is it possible to switch the template such that when the page has finished its animation it does not have any rendering transforms applied to it?
